Assets/Scripts/Augment/AugmentImplementations/DDRBodyAnimator.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DDRBodyAnimator : AugmentAnimator
+{
+    private Animator animator;
+    public override void OnFire(int remainingAmmo)
+    {
+        animator.SetTrigger("Fire");
```

Comment:
What does it do on fire? What does it even mean in this context? OnFire seems to be called on every beat in DDRBody, which is _more than a little_ confusing. The `BarrelAnimator` class wasn't really meant for bodies in the first place :/

Reply:
While I agree the trigger name itself ("Fire") might be a bit misleading (It's animating the membrane to vibrate when doing many gun functions, and to the beat), I think we should have a general augmentAnimator class

Reply:
I updated the name to something less vague than "Fire", which I think will suffice for now